src/main/resources/transformation/metafix-wip.fix

```diff
@@ -0,0 +1,227 @@
+  /*
+	<entity name="PersonCreator[]" flushWith="record">
+	*/
+set_array("PersonCreator[]")
+
+# Another bind would be `do with`
+do list(path:"700[013] ", "var": $i)
+
+
+
+/*
+    <entity name="" flushWith="700[013] " sameEntity="true">
+      <if>
+        <any flushWith="700[013] ">
+          <none flushWith="700[013] " sameEntity="true">
+            <data source="700[013] .[4e]" />
+          </none>*/
+	unless exists ("$i.[4e]")
+        /*  <data source="700[013] .[4e]">
+            <regexp match="^([Aa]ut|[Cc]re)" />
+          </data>
+        </any>
+      </if>
+      <call-macro name="personName"       field="700" />
+	  <macro name="personName">
+    <data name="$[name]" source="$[field][013] .a">
+      <replace pattern="(?&lt;!\p{Upper})\.$|[,]$" with=""
+			>
+    </data>
+  </macro>*/
+		replace_all("$i.a", "(?<!\\p{Upper})\\.$|[,]$", "")
+		copy_field("$i.a", "PersonCreator[].$append.personName")
+    /*  <call-macro name="personNumbering"  field="700" />
+  <macro name="personNumbering">
+    <data name="$[name]" source="$[field][013] .b">
+      <replace pattern="[,]$" with="" />
+    </data>
+  </macro>
+
+		*/
+		replace_all("$i.b", "[,]$", "")
+		copy_field("$i.b", "PersonCreator[].$last.personNumbering")
+    /*   <call-macro name="personTitle"      field="700" />
+		  <macro name="personTitle">
+    <entity name="$[name][]" flushWith="$[field][013] " sameEntity="true">
+      <data name="" source="$[field][013] .c">
+        <replace pattern="[,]$" with="" />
+      </data>
+    </entity>
+  </macro>
+				*/
+    #Since this is very nested and an array i tried an workaround. Does not work.
+		set_array("PersonCreator[].$last.personTitle[]")
+		do list(path: "$i.c", "var":"$j")
+      if exists("PersonCreator[].$last.@personTitle_array")
+        replace_all("$j", "[,]$", "")
+        copy_field("$j", "PersonCreator[].$last.personTitle[]")
+      else
+        add_field("PersonCreator[].$last.@personTitle_array","true")
+        replace_all("$j", "[,]$", "")
+        copy_field("$j", "PersonCreator[].$last.personTitle[].$append")
+      end
+		end
+    # This seems not to work too. Perhaps because of the $last?
+    remove_field("PersonCreator[].$last.@personTitle_array","true")
```

Comment:
BTW: This would also remove a field `true` if it existed. `remove_field(...)` removes all fields listed as params.

Reply:
> isn't the list finished first? At the end there is no need for that field anymore and PersonCreator is kept, therefore I need to delete this field.

There is no field named `true`, is there?

Reply:
you are right!